content/en/examples/application/guestbook/frontend-deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: php-redis
-        image: gcr.io/google_samples/gb-frontend:v5
+        image: corelab/gb-frontend:v5
```

Comment:
I get that we'd like to suggest an alternative.

- Why is this container image a good choice / could we switch to another image (maybe one that we, the 
Kubernetes project, control)?
- please specify the registry; `corelab` is not a DNS name that resolves.
  _I_ know you mean Docker Hub, but some readers might not.

Reply:
A better alternative may be us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5, which is now the image used in [the original tutorial.](https://cloud.google.com/kubernetes-engine/docs/tutorials/guestbook). Pasting the link in a browser leads to a Google cloud container registry.

A note could also be added at the top of the .yaml file directing readers to the [original tutorial's yaml file](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/HEAD/quickstarts/guestbook/frontend-deployment.yaml) if they have run into any trouble with the container image.

If this works, I will update the PR accordingly.

Reply:
@azn-abel I like this option. If we can get the page corrected, it will help resolve many duplicate issues which have been logged for this.

#44415 #44224 #44277

Reply:
Sounds good. I've corrected the page in a new commit.

Reply:
@azn-abel Looks good to me. However I don't have the permissions to add the lgtm label. Can you please request one of the other reviewers to help?

Reply:
Thanks - I've squashed the commits. I'll request @sftim for the lgtm label.